chasm/registrable_component.go

```diff
@@ -4,20 +4,21 @@ import (
 	"fmt"
 	"reflect"
 
-	enumspb "go.temporal.io/api/enums/v1"
+	"github.com/dgryski/go-farm"
```

Comment:
Wondering if it'd be worth using https://pkg.go.dev/hash/maphash which is in the standard library and would give you similar distribution or even `fnv` if we need support for 32 bit ints.

Reply:
Guess for now I still want to stick to go-farm which is used throughout the codebase for hashing purpose and avoid an exception.